tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -780,7 +780,7 @@ public function provide70Cases()
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```

Comment:
Sorry I don't understand what should I test.
Should or shouldn't multiline comments be fixed?
If yes, the closing tag MUST be fixed according to opening tag, and `*` must be aligned.

Maybe you didn't see the complete diff:
```diff
     /*
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```
Here that pipes are wrong aligned, not the `*`